app/src/main/kotlin/com/mapzen/erasermap/view/RouteModeView.kt

```diff
@@ -50,9 +50,6 @@ class RouteModeView : LinearLayout, RouteViewController, ViewPager.OnPageChangeL
     val mapListToggle: MapListToggleButton by lazy {
         findViewById(R.id.map_list_toggle)as MapListToggleButton
     }
-    val routeCancelButton: ImageView by lazy {
```

Comment:
Per discussion with @souperneon IRL I'm going to restore route cancel button (map view only) and it will be behave same as the system back button.